### ISSUE TITLE
Return domain of the authenticated user in PAM environment

### ISSUE THREAD

**Issue by pbrezina** [pbrezina/sssd-test#2480]
Cloned from Pagure issue: https://pagure.io/SSSD/sssd/issue/2476
- Created at 2014-10-24 13:14:42 by [adelton](https://pagure.io/user/adelton)
- Closed at 2020-03-24 14:07:13 as wontfix
- Assigned to [mzidek](https://pagure.io/user/mzidek)
---

When user is authenticated via pam_sss, any of the configured sssd domain would have been used. Having `alice` authenticated could mean it's `alice@example.net` or `alice@organization.org` or `alice@company.com`.

It'd be useful if the domain that was used to authenticate the user was available to the caller upon successful authentication / access check.

I propose `pam_putenv` of `AUTH_DOMAIN` value to be added.

### Comments

---

**Comment from [adelton](https://pagure.io/user/adelton) at [2014-11-06 10:35:36](https://pagure.io/SSSD/sssd/issue/2476#242292)**

Fields changed

milestone: NEEDS_TRIAGE => SSSD 1.13 beta

---

**Comment from [adelton](https://pagure.io/user/adelton) at [2014-11-06 10:36:07](https://pagure.io/SSSD/sssd/issue/2476#242293)**

Fields changed

blocking:  => 2151

---

**Comment from [adelton](https://pagure.io/user/adelton) at [2014-11-19 18:49:48](https://pagure.io/SSSD/sssd/issue/2476#242294)**

Fields changed

rhbz:  => todo

---

**Comment from [adelton](https://pagure.io/user/adelton) at [2015-02-10 16:14:51](https://pagure.io/SSSD/sssd/issue/2476#242295)**

Let's revive when we resume the work on the containerized sssd.

milestone: SSSD 1.13 beta => SSSD 1.13 backlog
priority: major => minor

---

**Comment from [adelton](https://pagure.io/user/adelton) at [2015-02-12 20:27:45](https://pagure.io/SSSD/sssd/issue/2476#242296)**

Mass-moving tickets not planned for the next two releases.

Please reply with a comment if you disagree about the move..

milestone: SSSD 1.13 backlog => SSSD 1.15 beta

---

**Comment from [adelton](https://pagure.io/user/adelton) at [2015-02-12 20:45:35](https://pagure.io/SSSD/sssd/issue/2476#242297)**

Actually after talking to Dmitri again, moving to 1.13 backlog, this might be useful for integration effort.

milestone: SSSD 1.15 beta => SSSD 1.13 backlog

---

**Comment from [adelton](https://pagure.io/user/adelton) at [2015-02-12 20:55:52](https://pagure.io/SSSD/sssd/issue/2476#242298)**

Re-setting priority of 1.13 backlog tickets used for planning.

priority: minor => major

---

**Comment from [adelton](https://pagure.io/user/adelton) at [2015-02-19 14:17:33](https://pagure.io/SSSD/sssd/issue/2476#242299)**

Jan requested this ticket sooner.

milestone: SSSD 1.13 backlog => SSSD 1.13 alpha

---

**Comment from [adelton](https://pagure.io/user/adelton) at [2015-03-24 11:52:25](https://pagure.io/SSSD/sssd/issue/2476#242300)**

Fields changed

rhbz: todo => 0

---

**Comment from [adelton](https://pagure.io/user/adelton) at [2015-06-08 19:22:50](https://pagure.io/SSSD/sssd/issue/2476#242301)**

Fields changed

owner: somebody => mzidek
sensitive:  => 0

---

**Comment from [adelton](https://pagure.io/user/adelton) at [2015-06-10 16:42:25](https://pagure.io/SSSD/sssd/issue/2476#242302)**

Fields changed

patch: 0 => 1

---

**Comment from [adelton](https://pagure.io/user/adelton) at [2015-06-18 16:08:29](https://pagure.io/SSSD/sssd/issue/2476#242303)**

Fields changed

milestone: SSSD 1.13 alpha => SSSD 1.13 beta

---

**Comment from [adelton](https://pagure.io/user/adelton) at [2015-09-21 16:39:48](https://pagure.io/SSSD/sssd/issue/2476#242304)**

Fields changed

milestone: SSSD 1.13.1 => SSSD 1.13.2

---

**Comment from [adelton](https://pagure.io/user/adelton) at [2015-10-11 22:08:23](https://pagure.io/SSSD/sssd/issue/2476#242305)**

This is important for the multitenancy case, but we're already trying to stabilize 1.13..moving to 1.14.

milestone: SSSD 1.13.2 => SSSD 1.14 beta

---

**Comment from [adelton](https://pagure.io/user/adelton) at [2016-02-16 13:56:40](https://pagure.io/SSSD/sssd/issue/2476#242306)**

Linked to Bugzilla bug: https://bugzilla.redhat.com/show_bug.cgi?id=1201262 (*Red Hat Enterprise Linux 7*)

rhbz: 0 => [https://bugzilla.redhat.com/show_bug.cgi?id=1201262 1201262]

---

**Comment from [adelton](https://pagure.io/user/adelton) at [2016-05-25 16:33:05](https://pagure.io/SSSD/sssd/issue/2476#242307)**

Because the bugzilla linked to this ticket was about sssd running in a container and this ticket is more about better support for Apache modules and similar 3rd party integration, I'm removing the bugzilla link.

Moreover, it looks like Jan doesn't need this ticket implemented in the immediate future, so I'm also moving the ticket to the 1.14 backlog.

milestone: SSSD 1.14 beta => SSSD 1.14 backlog
rhbz: [https://bugzilla.redhat.com/show_bug.cgi?id=1201262 1201262] => todo

---

**Comment from [adelton](https://pagure.io/user/adelton) at [2016-12-15 11:00:56](https://pagure.io/SSSD/sssd/issue/2476#242308)**

Since the 1.14 branch is transitioning into maintenance mode and new functionality is being developed in master which will become 1.15 eventually, I'm mass-moving tickets from the 1.14 backlog milestone to the "Future releases" milestone.

milestone: SSSD 1.14 backlog => SSSD Future releases (no date set yet)

---

**Comment from [adelton](https://pagure.io/user/adelton) at [2017-02-24 15:06:30](https://pagure.io/SSSD/sssd/issue/2476#242309)**

**Metadata Update from @adelton**:
- Issue assigned to mzidek
- Issue marked as blocked by: #2151
- Issue set to the milestone: SSSD Future releases (no date set yet)

---

**Comment from [adelton](https://pagure.io/user/adelton) at [2020-03-13 12:23:29](https://pagure.io/SSSD/sssd/issue/2476#633114)**

**Metadata Update from @thalman**:
- Custom field blocking reset (from 2151)
- Custom field design_review reset (from 0)
- Custom field mark reset (from 0)
- Custom field patch adjusted to on (was: 1)
- Custom field review reset (from 0)
- Custom field sensitive reset (from 0)
- Custom field testsupdated reset (from 0)
- Issue close_status updated to: None
- Issue tagged with: Canditate to close

---

**Comment from [adelton](https://pagure.io/user/adelton) at [2020-03-24 14:07:11](https://pagure.io/SSSD/sssd/issue/2476#636567)**

Thank you for taking time to submit this request for SSSD. Unfortunately this issue was not given priority and the team lacks the capacity to work on it at this time.

Given that we are unable to fulfill this request I am closing the issue as wontfix.

If the issue still persist on recent SSSD you can request re-consideration of this decision by reopening this issue. Please provide additional technical details about its importance to you.

Thank you for understanding.

---

**Comment from [adelton](https://pagure.io/user/adelton) at [2020-03-24 14:07:16](https://pagure.io/SSSD/sssd/issue/2476#636568)**

**Metadata Update from @pbrezina**:
- Issue close_status updated to: wontfix
- Issue status updated to: Closed (was: Open)
